### DISCUSSION TITLE
Is this an abandoned project?

### DISCUSSION THREAD

**Discussion by SeRi0uS007** [ValvePython/steam#470] [General]
How recently has Pull Requests been reviewed by the author? I have some ideas for extending and updating this library, but I don't know where to send Pull Requests...

**Comment by Marakai** [comment]
It certainly looks like it. I dug out an old project from some years ago, to update it and absolutely nothing works anymore, that relies on this. All errors are coming from within this project. There are so many PRs unmerged that could fix it that it'd take a full time maintainer to sort it all out.
